### PR TITLE
Welcome page links update to hlf from cloud

### PR DESCRIPTION
**Pull request by selvaprakash92**
Signed-off-by: selvaprakash92 <selvaprakash92@gmail.com>

#### Type of change

- Bug fix

#### Description

 Welcome page links update to hlf from cloud

### CODE REVIEW

File: packages/apollo/src/App.js [hunk 1]
```diff
@@ -202,7 +202,7 @@ class App extends Component {
 			default:
 				break;
 		}
-		let docPrefix = 'https://www.ibm.com/docs/en/blockchain-platform/2.5.3';
+		let docPrefix = 'https://www.ibm.com/docs/en/hlf-support/1.0.0';
```
Comment (dshuffma-ibm) [inline packages/apollo/src/App.js:205]: this line shouldn't be changed, please undo

the hfl-support vs blochain-platform switching happens on line 207 if the settings are correct

**Review by dshuffma-ibm — CHANGES_REQUESTED** [review]
minor change needed

File: packages/apollo/src/App.js (hunk 1 above)
Comment (selvaprakash92) [reply packages/apollo/src/App.js]: updated the code as per comment

**Review by dshuffma-ibm — APPROVED** [review]
thanks

**Pull request merged**